subprojects/scala/src/main/java/org/gradle/api/plugins/scala/ScalaBasePlugin.java

```diff
@@ -76,6 +77,13 @@
     @VisibleForTesting
     public static final String ZINC_CONFIGURATION_NAME = "zinc";
     public static final String SCALA_RUNTIME_EXTENSION_NAME = "scalaRuntime";
+    /**
+     * Configuration for scala compiler plugins.
+     *
+     * @since 6.1
```

Comment:
This will need to be updated to 6.3 or 6.4 depending on when it gets ready to be merged.